build_forge.gradle

```diff
@@ -117,9 +117,9 @@ dependencies {
     bootstrap(libs.securemodules) // Has Union file system in it
     bootstrap(libs.unsafe)        // Needed by securemodules
     bootstrap(libs.bundles.asm)   // Needed by securemodules
-    
+
     implementation(libs.jopt.simple) { version { strictly '5.0.4' } }
```

Comment:
We HAVE to do something about this jopt simple library issue. I'm willing to investigate. What library is causing this? Would it be possible to use a gradle plugin to force this? If we have to tack it on to FG6, I can.

Reply:
Actually, looking at it, we dont need this anymore. I think we've got all the transitives down, but that isnt related to this PR

Reply:
Ok. Let me know if anything changes so I can look into it.